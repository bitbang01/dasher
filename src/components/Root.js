import glamorous from 'glamorous'
import { fonts } from '../theme'

const Root = glamorous.div({
  fontFamily: fonts.sans,
  boxSizing: 'border-box',

  ' *, *:before, *:after': {
    boxSizing: 'inherit',
  },
})

export default Root
