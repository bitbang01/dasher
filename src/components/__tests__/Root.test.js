import React from 'react'
import renderer from 'react-test-renderer'

import Root from '../Root'

it('renders without crashing', () => {
  const tree = renderer.create(<Root />).toJSON()
  expect(tree).toMatchSnapshot()
})
