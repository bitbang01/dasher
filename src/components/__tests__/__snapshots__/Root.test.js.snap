// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
.css-0,
[data-css-0] {
  font-family: system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
  box-sizing: border-box;
}

.css-0 *,
[data-css-0] *,
.css-0 *:before,
[data-css-0] *:before,
.css-0 *:after,
[data-css-0] *:after {
  box-sizing: inherit;
}

<div
  className="css-0"
/>
`;
